cms/methodology/models.py

```diff
@@ -0,0 +1,143 @@
+    parent_page_types: ClassVar[list[str]] = ["topics.TopicPage"]
+
+    template = "templates/pages/methodology_page.html"
+
+    summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    publication_date = models.DateField()
+    last_revised_date = models.DateField(blank=True, null=True)
+
+    contact_details = models.ForeignKey(
+        "core.ContactDetails",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    content = StreamField(SectionStoryBlock())
+
+    show_cite_this_page = models.BooleanField(default=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        MultiFieldPanel(
+            [
+                FieldRowPanel(
+                    [
+                        FieldPanel(
+                            "publication_date",
+                        ),
+                        FieldPanel("last_revised_date"),
+                    ]
+                ),
+                FieldPanel("contact_details"),
+                FieldPanel("show_cite_this_page"),
+            ],
+            heading=_("Metadata"),
+            icon="cog",
+        ),
+        FieldPanel("content", icon="list-ul"),
+        InlinePanel("related_pages", label="Related publications"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("summary"),
+        index.SearchField("content"),
+    ]
+
+    def clean(self) -> None:
+        """Additional validation on save."""
+        super().clean()
+
+        if self.last_revised_date and self.last_revised_date <= self.publication_date:
+            raise ValidationError({"last_revised_date": _("The last revised date must be after the published date.")})
+
+    def get_context(self, request: HttpRequest, *args: Any, **kwargs: Any) -> dict:
+        """Additional context for the template."""
+        context: dict = super().get_context(request, *args, **kwargs)
+        context["table_of_contents"] = self.table_of_contents
+        context["related_publications"] = self.get_formatted_related_publications_list(request=request)
+        return context
+
+    @cached_property
+    def related_publications(self) -> "PageQuerySet":
+        """Return a `PageQuerySet` of items related to this page via the
+        `PageRelatedPage` through model, and are suitable for display.
+        The result is ordered to match that specified by editors using
+        the 'page_related_pages' `InlinePanel`.
+        """
+        # NOTE: avoiding values_list() here for compatibility with preview
+        # See: https://github.com/wagtail/django-modelcluster/issues/30
+        ordered_page_pks = tuple(item.page_id for item in self.related_pages.all().only("page"))
+        return order_by_pk_position(
+            Page.objects.live().public().specific(),
```

Comment:
```suggestion
            AnalysisPage.objects.live().public().specific(),
```
Question to @zerolab, wouldn't it be more efficient if we queried here the AnalysisPage table instead?
Are there any downsides of that? 
It ties the Methodology page with the Analysis page, but we have that relationship explicit in the requirements so I think that'd be fine.

Reply:
These are more or less equivalent, but the current approach allows to scale should we allow more than analysis pages